### PR TITLE
fileFormat additions for PEC

### PR DESCRIPTION
**Pull request by kelshmo**

### CODE REVIEW

File: synapseAnnotations/data/sageCommunity.json [hunk 1]
```diff
@@ -519,6 +519,21 @@
         "value": "mtx",
         "description": "Matrix Market Exchange Format",
         "source": "https://math.nist.gov/MatrixMarket/formats.html#MMformat"
+      },
+      {
+        "value":"bfilt",
+        "description": "contains the final ATACSeq peaks after removing the encode blacklist regions.",
+        "source":"https://sites.google.com/site/anshulkundaje/projects/blacklists"
```
Comment (karawoo) [inline synapseAnnotations/data/sageCommunity.json]: Is this really a file format, or are [these files](https://github.com/Boyle-Lab/Blacklist/tree/master/lists) (are they going to be hosted on Synapse?) just BED files with specific contents?

File: synapseAnnotations/data/sageCommunity.json [hunk 2]
```diff
@@ -519,6 +519,21 @@
...
+      },
+      {
+        "value":"tagAlign",
+        "description":"Tag Alignment provides genomic mapping of short sequence tags.",
+        "source":"https://genome.ucsc.edu/FAQ/FAQformat.html#format15"
+      },
+      {
+        "value":"dup",
```
Comment (karawoo) [inline synapseAnnotations/data/sageCommunity.json]: I'm guessing SAM/BAM aren't adequate for these files?

File: synapseAnnotations/data/sageCommunity.json (hunk 1 above)
Comment (kelshmo) [reply synapseAnnotations/data/sageCommunity.json]: The file path is actually .bfilt (for example, https://www.synapse.org/#!Synapse:syn18359515)

File: synapseAnnotations/data/sageCommunity.json (hunk 2 above)
Comment (kelshmo) [reply synapseAnnotations/data/sageCommunity.json]: Also, another strange file path .dup (https://www.synapse.org/#!Synapse:syn18359507). This output is the reprocessing pipeline for PEC - so we will continue to produce files with these file paths.

File: synapseAnnotations/data/sageCommunity.json (hunk 1 above)
Comment (karawoo) [reply synapseAnnotations/data/sageCommunity.json]: The file format and file extension are not necessarily the same thing, so I guess I'm trying to understand what about these files warrants its own format. Is there something about the structure or format of these files that is unique and standardized?

File: synapseAnnotations/data/sageCommunity.json (hunk 2 above)
Comment (kelshmo) [reply synapseAnnotations/data/sageCommunity.json]: tagAlign is a specific form of a bed file, dup is a SAM or BAM file with additional notation for duplicate reads...Maybe the files could be more broadly characterized, but why not be more granular and point to the specific documentation?

File: synapseAnnotations/data/sageCommunity.json (hunk 1 above)
Comment (kelshmo) [reply synapseAnnotations/data/sageCommunity.json]: In the case of the bfilt, the file structure does not appear to deviate from a bed file so I will remove value.

File: synapseAnnotations/data/sageCommunity.json (hunk 2 above)
Comment (karawoo) [reply synapseAnnotations/data/sageCommunity.json]: yeah that seems fine if they are different in a standardized way. my main concern was with bfilt

**Comment by kelshmo** [comment]
shoot can I unmerge

**Pull request merged**